tests/dex/dex_trades_assert_freshness.sql

```diff
@@ -56,7 +56,7 @@ with delays as (
     SELECT
         project
         , blockchain
-        , int(now() - max(block_time))/3600 as age_of_last_record_hours
+        , date_diff('hour', max(block_time), now()) as age_of_last_record_hours
```

Comment:
What do we think of this @jeff-dude @antonio-mendes. The previous line was throwing errors on dunesql

Reply:
I think your proposal works well @couralex6 ! Just tested with a few different timestamps and it's accurate